README.md

```diff
@@ -1,24 +1,36 @@
-# README
+# Blog-app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is a an Blog app build using Ruby on Rails Where you can sign up, create your own posts and comment and like other users posts
+## Built With
 
-Things you may want to cover:
+- ![](https://img.shields.io/badge/Github-blueviolet)
+- ![](https://img.shields.io/badge/Ruby-red)
+- ![](https://img.shields.io/badge/Ruby*on*Rails-red)
 
-* Ruby version
+## Authors
 
-* System dependencies
+👤 **ibrahim naseer**
```

Comment:
- [ ] Hello and good work on the app, but please add relevant instructions on how to clone and run the app.